test/scenarios/cos9-src@rpm-standard1.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```

Comment:
Do you think we need 2 COS9 scenarios in presubmit? We don't even support COS9. I'd move one of them to periodics and remove other one, so that we have some coverage of this new functionality.

Reply:
I agree that we should be flexible with this. 
It's mostly about testing that MicroShift install and startup work. I'll implement the suggestion.